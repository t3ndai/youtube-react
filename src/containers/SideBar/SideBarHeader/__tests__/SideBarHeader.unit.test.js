import React from "react";
import { shallow } from "enzyme";
import { SideBarHeader } from "../SideBarHeader";

test("it renders SideBarHeader", () => {
  const wrapper = shallow(<SideBarHeader />);
  expect(wrapper).toMatchSnapshot();
});
