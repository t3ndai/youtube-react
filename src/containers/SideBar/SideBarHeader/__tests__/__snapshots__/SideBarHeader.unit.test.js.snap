// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders SideBarHeader 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  />
</MenuItem>
`;
